resources/js/Shared/TextDuration.vue

```diff
@@ -58,49 +58,54 @@ input[type=number] {
         min.
       </span>
 
-      <the-mask v-model="localMinutes"
+      <the-mask :value="proxyMinutes"
                 mask="##"
                 class="br2 f5 pt2 pb0 ph1 outline-0 di tc bg-white"
                 type="text"
                 :masked="false"
                 placeholder="00"
                 :data-cy="datacy + '-minutes'"
-                @input="broadcastTotal()"
+                @input="proxyMinutes = $event.target.value"
       />
     </div>
   </div>
 </template>
 
 <script>
-import {TheMask} from 'vue-the-mask';
+import TheMask from 'vue-the-mask/src/component';
 
 export default {
 
-  components: {TheMask},
-  inheritAttrs: false,
+  components: {
+    TheMask
+  },
+  //  inheritAttrs: false,
+
+  model: {
+    prop: 'modelValue',
+    event: 'update:modelValue'
+  },
 
   props: {
     id: {
       type: String,
-      default() {
-        return `text-input-${this._uid}`;
-      },
+      default: 'text-input-',
     },
     errors: {
       type: String,
       default: '',
     },
-    total: {
+    modelValue: {
       type: Number,
       default: 0,
     },
     hours: {
       type: Number,
-      default: 0,
+      default: undefined,
```

Comment:
@djaiss I think it looks better if the default is not filled, it's a little behavior change.

Reply:
I agree.
Although I don't know if it's related, but when you key-up in this widget now, it triggers the following error:
```
Uncaught TypeError: $event.target is undefined
    0]!./node_modules/vue-loader/dist/templateLoader.js??ruleSet[1].rules[2]!./node_modules/vue-loader/dist/index.js??ruleSet[0].use[0]!./resources/js/Shared/TextDuration.vue?vue&type=template&id=59f368bd&scoped=true/render</_cache[1 TextDuration.vue:48
    callWithErrorHandling runtime-core.esm-bundler.js:154
    callWithAsyncErrorHandling runtime-core.esm-bundler.js:163
    emit runtime-core.esm-bundler.js:642
    refresh component.vue:64
    onInput component.vue:56
    1 component.vue:2
    patchStopImmediatePropagation runtime-dom.esm-bundler.js:347
    callWithErrorHandling runtime-core.esm-bundler.js:154
    callWithAsyncErrorHandling
```

Reply:
OK I found what was wrong and I've improved the reliability, for instance to store a change once only.